examples/basics/basics/documented_workflow.py

```diff
@@ -63,6 +63,18 @@ def sphinx_docstring(df: pd.DataFrame, data: PandasData = PandasData()) -> pd.Da
     """
     return add_data(df=df, data=data)
 
+# %% [markdown]
+# You'll see the short description in the workflow page.
+# :::{figure} https://i.ibb.co/CtJV7Gq/Screenshot-2023-07-22-at-12-16-38-AM.png
+# :alt: short description
+# :class: with-shadow
+# :::
+#
+# If you click the workflow, you'll see the long description in the basic information section.
+# :::{figure} https://i.ibb.co/1QPtJNK/Screenshot-2023-07-22-at-12-20-50-AM.png
```

Comment:
Can we upload it to the repo static resources?

Reply:
https://github.com/flyteorg/static-resources/pull/23

Reply:
done